site/package.json

```diff
@@ -40,13 +40,13 @@
     "@tanstack/react-query": "^5.28.4"
   },
   "devDependencies": {
-    "contentlayer": "0.2.9",
-    "next-contentlayer": "0.2.9",
+    "contentlayer": "0.3.4",
+    "next-contentlayer": "0.3.4",
     "next-sitemap": "^4.2.3"
   },
   "scripts": {
     "dev": "next dev --port 3001",
-    "build": "INIT_CWD=$PWD next build",
+    "build": "contentlayer build && INIT_CWD=$PWD next build",
```

Comment:
Needed since it has some problems on pages router. More context [here](https://github.com/contentlayerdev/contentlayer/issues/415)